packages/showcases/react/stories/components/VtmnTextInput/VtmnTextInput.stories.tsx

```diff
@@ -30,6 +31,15 @@ export default {
       defaultValue: 'Helper text goes here',
       control: { type: 'text' },
     },
+    icon: {
```

Comment:
What do you think about rename it to "iconValue" instead of "icon"?
It could clearer for user to add a string instead of a `VtmnIcon` Component inside this prop no?

Reply:
@gabbloquet ?

Reply:
I think it's overkill because the only icon props its that one.
It could be interesting to have other advice on it, @RomainJoly , @gaspard-mathon, what's your advice on it ?
`icon` VS `iconValue` props ?

Reply:
Yes I understand. But when I see "icon" for prop name, I'm waiting for passing a `<Icon>` element as prop instead of a value

Reply:
For me it is ok to have props name "icon".The prop name "icon" in the button component of Vitamin is waitting a string.

Reply:
I don't see any problem to call the prop "icon". The consumers can see what type of data is required in the showcase.

Reply:
Okay thanks @RomainJoly !